.gitignore

```diff
@@ -45,3 +45,8 @@ _build
 .idea
 .vscode
 *~
+
+# virtualenv
+Pipfile
+Pipfile.lock
+Makefile
```

Comment:
If you're interested in adding this to this file, please submit that as an issue to our cookiecutter [here](https://github.com/adafruit/cookiecutter-adafruit-circuitpython/issues) so it can be discussed. Thanks!

Reply:
removed